spec/models/food_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
```

Comment:
- Hi, Please Your Test is giving errors at my end. Try to look at it and send the screenshot. the next reviewer gonna confirm. I tried to update my Ruby version but still, have the errors. 

- 
![test](https://user-images.githubusercontent.com/63972670/187648393-de9ad935-d597-4885-a04a-0a04aa30931c.JPG)

Reply:
- Hi. Your App Cannot Run without Migration. Can you see the screenshot I posted? This shows that I ran `db migrate` and run your App. 
- If you are sure about it, then move on, solve the required change above and submit it again. Sorry in advance **IF** the mistake is on my end.